pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -477,10 +477,14 @@ class Build {
                         def rerunIterations = '3'
                         def fipsTestBuildSuffix = "";
                         def buildList = ""
+                        // testType value examples: 
+                        // extended.functional.fips140_3_OpenJCEPlusFIPS.FIPS140-3
+                        // sanity.jck.fips140_3_OpenJCEPlusFIPS
                         if ("${testType}".contains(".fips")) {
                             String[] tokens = testType.split('\\.')
-                            testType = tokens[0] + "." + tokens[1]
-                            fipsTestBuildSuffix = tokens[2]
+                            def levelGroup = tokens[0] + "." + tokens[1]
+                            fipsTestBuildSuffix = testType.replace(levelGroup + ".", "")
```

Comment:
Can you add the fipsTestBuildSuffix result to the comment above? It looks like this is stripping off `<level>.<group>.` from the testType. Is that correct? ie. Taking everything after `level.group.`.

Reply:
yes, it is correct. `level.group.` will be removed from `testType ` (i.e., `extended.openjdk.fips140_3_OpenJCEPlusFIPS.FIPS140-3`). In this case, fipsTestBuildSuffix should be `fips140_3_OpenJCEPlusFIPS.FIPS140-3`.